examples/imgui_impl_x11.cpp

```diff
@@ -34,6 +34,8 @@ static xcb_key_symbols_t*   g_KeySyms;
 static timespec             g_LastTime;
 static timespec             g_CurrentTime;
 
+static bool                 g_HideXCursor = false;
```

Comment:
With Win32 we just call `SetCursor(NULL)` every frame we want the mouse cursor hidden. With X we have to do a request/reply for that. While I don't think that's a huge issue doing that every frame, I observed how various other 3D context X windows behave. I couldn't find any that did a request/reply every frame, every window I observed did not have any events when the window is idle. So while this seems silly, I can't figure out any other way on checking 'did the state of the MouseDrawCursor flag change?' without keeping a copy and comparing between frames.

Reply:
You should keep a copy indeed (of the ImGuiMouseCursor value) if you want to detect change. When the value is _None that means to hide the cursor.